### ISSUE TITLE
Compile warning: var devcards.core/test-doc is not public

### ISSUE THREAD

**Issue by n2o** [bhauman/devcards#157]
Hey, I am experimenting with dc/deftest and tried to implement some tests directly in devcards. Previously I wrote tests with lein doo, but this is not really satisfying.

However, I am getting the error message that test-doc is not public. I am not quite sure, why this error appears. I have many devcards in this project and they are all operating well. Also the other tests are all okay.

```clojure
(ns some.namespace
   (:require
             [cljs.test :refer [testing is are]]
             [devcards.core :as dc]))
 
 (dc/deftest foo
 ^--- var: devcards.core/test-doc is not public
   "Sample card"
   (testing "Fix me, I fail"
     (is false)))
```

Some of my dependencies:
```edn
[org.clojure/clojure "1.10.0"]
[org.clojure/clojurescript "1.10.520"]
[devcards "0.2.6"]
[com.bhauman/figwheel-main "0.2.0"]
[com.bhauman/rebel-readline-cljs "0.1.4"]
```

Thanks for your help. I have no idea what could be wrong. I am using devcards in several projects and i love it, but this is a strange error message.

**Comment by bhauman** [comment]
This is probably because you are using the latest clojurescript.

Are you familiar with cljs-test-display? 
https://github.com/bhauman/cljs-test-display

You should be using it for tests instead of devcards.

**Comment by n2o** [comment]
I am now using cljs-test-display, which is a great alternative, yes. Although it was hard to follow your instructions in your README. Had to combine it with instructions from the figwheel-main documentation.

Okay, I can use this for testing, thanks. But still, your library provides the deftest cards.

**Comment by danielstockton** [comment]
Also getting `devcards.core/dom-node* is not public` with latest cljs.
